derive_builder/tests/derive_trait.rs

```diff
@@ -0,0 +1,21 @@
+#[macro_use]
+extern crate derive_builder;
+
+#[derive(Debug, Default, Clone)]
+struct NotPartialEq(String);
+
+#[derive(Debug, Clone, Builder)]
+#[builder(derive(Debug, PartialEq, Eq))]
```

Comment:
I actually have that; the `defaults` test down below will fail if `LoremBuilder` doesn't implement `PartialEq` and `Debug`. I've made this more explicit with a comment in the test.